config/archive_test.go

```diff
@@ -0,0 +1,120 @@
+package config
+
+import (
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/contiv/volplugin/watch"
+	. "gopkg.in/check.v1"
+)
+
+const (
+	testPolicyName = "foobar"
+)
+
+func revisionCount(s *configSuite) int {
+	revisions, _ := s.tlc.ListPolicyRevisions(testPolicyName)
+	return len(revisions)
+}
+
+func createTestPolicy(s *configSuite, c *C) {
+	err := s.tlc.CreatePolicyRevision(testPolicyName, "{}")
+	c.Assert(err, IsNil)
+}
+
+func (s *configSuite) TestCreatePolicyRevision(c *C) {
+	c.Assert(revisionCount(s), Equals, 0)
+	createTestPolicy(s, c)
+	c.Assert(revisionCount(s), Equals, 1)
+}
+
+func (s *configSuite) TestListPolicyRevisions(c *C) {
+	c.Assert(revisionCount(s), Equals, 0)
+
+	// edge case: the second may roll over between saving the current timestamp
+	//            and the HTTP request to upload the policy being handled
+	t := int(time.Now().Unix())
```

Comment:
Sure, I don't think verifying the timestamp is exact really matters for this test.  The TestWatchForPolicyChanges() test below tests that since it has to verify that the stdout output was correct.